frontend/README.md

```diff
@@ -10,13 +10,15 @@ Create a .env file and populate it with this.
 NODE_ENV=development
 API_URL=http://backend:8080/
 CHOKIDAR_USEPOLLING=true
+REACT_APP_SITE_MINDER_LOGOUT_URL=https://logontest.gov.bc.ca/clp-cgi/logoff.cgi or https://logon7.gov.bc.ca/clp-cgi/logoff.cgi
```

Comment:
Siteminder is a whole word - please change to `REACT_APP_SITEMINDER_...`

Reply:
You right